cfgov/v1/tests/test_admin_views.py

```diff
@@ -0,0 +1,119 @@
+from django.core.urlresolvers import reverse
+from django.test import TestCase, override_settings
+from django.contrib.auth.models import User, Permission, Group
+from django.contrib.contenttypes.models import ContentType
+
+import mock
+
+
+cache_config = {'BACKEND': 'v1.models.akamai_backend.AkamaiBackend',
+                'CLIENT_TOKEN': 'fake',
+                'CLIENT_SECRET': 'fake',
+                'ACCESS_TOKEN': 'fake'}
+
+
+# This is so that certain tests will function when migrations are not run
```

Comment:
This is a minor thing, but I think typically a description of the function lives in a docstring underneath the function name, with triple quotes, rather than as a comment.